obspy/io/seiscomp/event.py

```diff
@@ -116,7 +116,7 @@ def _write_sc3ml(catalog, filename, validate=False, verbose=False,
     nsmap_ = getattr(catalog, "nsmap", {})
     quakeml_doc = Pickler(nsmap=nsmap_).dumps(catalog)
     xslt_filename = os.path.join(os.path.dirname(__file__), 'data',
-                                 'quakeml_1.2__sc3ml_0.9.xsl')
+                                 'quakeml_1.2__sc3ml_0.10.xsl')
```

Comment:
No idea.. do we want to be able to write different versions of SC3ML? Probably not and writing the most recent version is OK to keep maintenance down, I guess..